snaps/reference/cli/options.md

```diff
@@ -7,542 +7,564 @@ description: See the Snaps CLI options reference.
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Snaps command line options
+# Snaps configuration options
 
-This reference describes the syntax of the Snaps command line interface (CLI) options.
+This reference describes the syntax of the Snaps command line interface (CLI) configuration options.
+You can specify these options in the
+[configuration file](../../learn/about-snaps/files.md#configuration-file).
 
-## Specify options
+## `bundler`
 
-You can specify options:
+<Tabs>
+<TabItem value="Syntax">
+
+```javascript
+bundler: <BUNDLER>,
+```
 
-- In the [configuration file](../../learn/about-snaps/files.md#configuration-file).
+</TabItem>
+<TabItem value="Example">
+
+```javascript
+bundler: "webpack",
+```
 
-- On the command line using the `yarn mm-snap` command.
+</TabItem>
+</Tabs>
 
-  ```bash
-  yarn mm-snap [SUBCOMMAND] [OPTIONS]
-  ```
+The bundler to use.
+The options are `"webpack"` and `"browserify"`.
+The default is `"webpack"`.
 
-## Options
+:::caution important
+We recommend using the Webpack bundler.
+The Browserify-based configuration is deprecated and will be removed in the future.
+This reference describes the configuration options for Webpack.
+For Browserify, see the
+[legacy options](https://github.com/MetaMask/snaps/tree/main/packages/snaps-cli#legacy-options).
```

Comment:
Maybe link to a commit, since we will remove these docs from the README at some point.
```suggestion
[legacy options](https://github.com/MetaMask/snaps/tree/455366f19281801ed4220431100e45237dd5cf1e/packages/snaps-cli#legacy-options).
```